docs/running.rst

```diff
@@ -22,12 +24,36 @@ To run on 4 machines with 4 GPUs each:
 
     $ horovodrun -np 16 -H server1:4,server2:4,server3:4,server4:4 python train.py
 
+You can also specify host nodes in a host file:
+For example,
+
+.. code-block:: bash
+
+    $ cat myhostfile
+
+aa slots=2
```

Comment:
If you'd like this to be part of the code block, you can indent lines 34-36 3 spaces